bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/ScriptTransformationService.java

```diff
@@ -203,4 +218,30 @@ private void disposeScriptEngine(ScriptEngine scriptEngine) {
             logger.trace("ScriptEngine does not support AutoCloseable interface");
         }
     }
+
+    @Override
+    public @Nullable Collection<ParameterOption> getParameterOptions(URI uri, String param, @Nullable String context,
```

Comment:
I'm a bit lost here why the `ScriptTranformationService` provides the configuration options for the profile - this somehow feels like a dependency, which isn't desired (removing the transformation service would impact the profile). Wouldn't the `ScriptProfileFactory` be the better place?